.github/workflows/issue_pm_ai.yml

```
@@ -0,0 +1,114 @@
+name: Issue PM AI
+
+on:
+  issues:
+    types: [opened]
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  respond:
+    # Only run for issues opened in the main repository (not forks)
+    if: github.repository == 'internetarchive/openlibrary'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Load PM AI Instructions
+        id: pm
+        run: |
+          {
+            echo "INSTRUCTIONS<<EOF"
+            cat .github/prompts/issue_pm_instructions.md
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
```

```suggestion
          } >> "$COPILOT_PROMPT"
```
`COPILOT_PROMPT` may be a better environment variable name.